### PR TITLE
updates verify gsp cluster egress ip addrs

### PR DESCRIPTION
**Pull request by chrisfarms**
egress IPs changed for verify gsp cluster changed after the
migration to the EKS based deployment a while back, this updates them so
we can wire up alertmanager again.

### CODE REVIEW

**Review by issyl0 — DISMISSED** [review]
I trust that these are correct - I don't think I have any way to check?

**Review by andy-paine — APPROVED** [review]
Assume these are correct too but ```Error running fmt: In ./terraform/projects/infra-security-groups-production/main.tf: At 91:5: error parsing list, expected comma or list end, got: STRING
The command "find . -name '*.tf' | xargs tools/terraform-format.sh" exited with 123.``` from Travis

File: terraform/projects/infra-security-groups-production/main.tf [hunk 1]
```diff
@@ -86,10 +86,10 @@ module "infra-security-groups" {
     "35.176.252.164/32",
 
     # verify gsp
-    "35.176.76.151/32",
+    "18.130.62.225"
```
Comment (issyl0) [inline terraform/projects/infra-security-groups-production/main.tf]:
```suggestion
    "18.130.62.225",
```

File: terraform/projects/infra-security-groups-production/main.tf [hunk 2]
```diff
@@ -86,10 +86,10 @@ module "infra-security-groups" {
...
 
-    "18.130.183.50/32",
-    "18.130.4.61/32",
+    "3.8.38.79"
```
Comment (issyl0) [inline terraform/projects/infra-security-groups-production/main.tf]:
```suggestion
    "3.8.38.79",
```

File: terraform/projects/infra-security-groups-production/main.tf [hunk 3]
```diff
@@ -86,10 +86,10 @@ module "infra-security-groups" {
...
+    "52.56.126.222"
```
Comment (issyl0) [inline terraform/projects/infra-security-groups-production/main.tf]:
```suggestion
    "52.56.126.222",
```

**Comment by philandstuff** [comment]
I suspect the IP addresses *must* end with `/32` or it will break 😞

**Review by issyl0 — APPROVED** [review]

**Pull request merged**